categories/categories/categories.php

```diff
@@ -83,6 +89,20 @@ public function getCategoriesList()
 		$model->setState('filter.component', $parts[0]);
 		$model->setState('filter.section', (count($parts) > 1) ? $parts[1] : null);
 
-		return $model->getItems();
+		$rows = $model->getItems();
+		$obj = new stdclass;
+		if (count($rows) > 0)
+		{			
+			$obj->success = true;
+			$obj->data = $rows;
+
+			return $obj;
+		}
+		else
+		{
+			$obj->success = false;
+			$obj->message = 'System does not have categories';
```

Comment:
@vinayak-patil Use language constant for text "System does not have categories"

Reply:
@ankush-maherwal Thank you for your guidance. Will revise and update you.